sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -487,15 +484,29 @@ export class StudentService extends RecordDataModelService<Student> {
       .innerJoin("student.sinValidation", "sinValidation")
       .where("user.isActive = true");
     if (institutionId) {
-      searchQuery.andWhere(
-        new Brackets((qb) => {
-          qb.where("offeringInstitution.id = :institutionId", {
-            institutionId,
-          }).orWhere("pirInstitution.id = :institutionId", { institutionId });
-        }),
-      );
+      searchQuery
```

Comment:
To fix the point about "Adjusted query to not get either Cancelled applications that weren't assessed or Draft applications;", is it not the case to have `.leftJoin("application.currentAssessment", "studentAssessment")` converted to a `innerJoin` and have it moved inside the `if (institutionId)`?

Reply:
mm,so just to refresh,  so if an application had an exception/cra validation and is in progress and the student cancels the application. shouldn't the student be seen when searched?

Reply:
Yes @ann-aot, but I do not see how my comment would change it. My suggestion would still return any application that had an assessment created (on Submit) even if the workflow was never executed.

Reply:
Yes, you're right. Also, we can do inner joins to location and institution.
They are already inside the `if(institutionId)`.

Reply:
no @andrewsignori-aot, it was not about the comment. it was to refresh the logic. Thanks 👍